src/processing.py

```diff
@@ -0,0 +1,233 @@
+import polars as pl
+import logging
+import sqlalchemy as sa
+from database import data_pipeline
+from datetime import datetime
+import os
+
+# Set up logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+# Set database path to the parent directory
+db_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'orders.db')
+
+
+def create_engine():
+    """Create database connection. If the database doesn't exist, create it and populate it from database.py"""
```

Comment:
I like descriptive and short docstrings like this! Kindly add this to some of your not-so-clear functions.